src/css/controls/imports/autostartmute.less

```diff
@@ -0,0 +1,23 @@
+@import "../../shared-imports/vars";
+@import "../../shared-imports/icons";
```

Comment:
There's one situation when it's used and that is on the error screen to display the error icon in the center of the player.  I was going to try removing that dependency by making the `jw-display-container` not display in controls: false